components/portal/widgets/controllers.js

```diff
@@ -424,6 +444,14 @@ define(['angular', 'moment'], function(angular, moment) {
           // Log a missing-config error
           $log.warn('An action item was missing one or '
             + 'more required configuration options');
+
+            if ($scope.config.actionItems[i].textPlural) {
```

Comment:
It relies on the `textPlural` to tell the user what errored, so yes that was intentional. However, rather than failing silently, should probably tell the user it couldn't count some unknown thing.